boards/arm/b_g474e_dpow1/board.c

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2022 The Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <devicetree.h>
+#include <drivers/gpio.h>
+#include <drivers/adc.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(board, LOG_LEVEL_DBG);
+
+#if DT_HAS_COMPAT_STATUS_OKAY(usbc_port)
+
+#define VBUS DT_PATH(vbus)
+#define LED_LABEL       DT_GPIO_LABEL(LED0_NODE, gpios)
+#define LED_PIN         DT_GPIO_PIN(LED0_NODE, gpios)
+#define LED_FLAGS       DT_GPIO_FLAGS(LED0_NODE, gpios)
+
+/* Common settings supported by most ADCs */
+#define ADC_RESOLUTION          12
+#define ADC_GAIN                ADC_GAIN_1
+#define ADC_REFERENCE           ADC_REF_INTERNAL
+#define ADC_ACQUISITION_TIME    ADC_ACQ_TIME_DEFAULT
+#define ADC_REF_MV              3300
+
+static const struct device *dev_adc;
+static int16_t sample_buffer;
+
+static const uint32_t output_ohm = DT_PROP(VBUS, output_ohms);
+static const uint32_t full_ohm = DT_PROP(VBUS, full_ohms);
+
+struct adc_channel_cfg channel_cfg = {
+	.gain = ADC_GAIN,
+	.reference = ADC_REFERENCE,
+	.acquisition_time = ADC_ACQUISITION_TIME,
+	.differential = 0
+};
+
+struct adc_sequence sequence = {
+	.channels = BIT(8),
+	.buffer = &sample_buffer,
+	/* buffer size in bytes, not number of samples */
+	.buffer_size = sizeof(sample_buffer),
+	.resolution = ADC_RESOLUTION,
+};
+
+/* usbc.rst board start */
+/**
+ * @brief This function is used to measure the VBUS in millivolts
+ */
+int board_vbus_meas(int *mv)
+{
+	int32_t value;
+	int ret;
+
+	ret = adc_read(dev_adc, &sequence);
+	if (ret != 0) {
+		printk("ADC reading failed with error %d.\n", ret);
+		return ret;
+	}
+
+	value = sample_buffer;
+	ret = adc_raw_to_millivolts(ADC_REF_MV, ADC_GAIN, ADC_RESOLUTION, &value);
+	if (ret != 0) {
+		printk("Scaling ADC failed with error %d.\n", ret);
+		return ret;
+	}
+
+	/* VBUS is scaled down though a voltage divider */
+	*mv = (value * 1000) / ((output_ohm * 1000) / full_ohm);
+
+	return 0;
+}
+
+/**
+ * @brief This function configures the board for USBC operation
+ */
+int board_config(void)
+{
+	dev_adc = DEVICE_DT_GET(DT_IO_CHANNELS_CTLR(VBUS));
+	if (!device_is_ready(dev_adc)) {
+		printk("ADC device not found\n");
```

Comment:
Seems like logging should be used instead, no? A general comment about this PR.

Reply:
Updated to use LOG_INF